docs/notebooks/analysis/swept_langmuir/find_floating_potential.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>"objec representing" -&gt; "object representing". Also, should vf and vf_err have units attached?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/889/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
🤔 I could give them units...

Reply:
Since the input `current` and `voltage` arrays are not conditioned for units, I don't think `vf` and `vf_err` should be either.  If we put units on `vf` and `vf_err`, then I think units should be enforced on `current` and `voltage`.  I'm between two minds on which way to go with this...

Reply:
This is why I'm in two minds about having units on this function. I like enforcing units because it creates less chances for improper use, but it will slow the function down.  Like the [distribution discussion](https://github.com/PlasmaPy/PlasmaPy/pull/816#discussion_r432489113), this function will be applied repeatedly to to a series of arrays and checking for units will significantly hamper the performance.  Thus, I went with no units to start.

Reply:
Okay, in that case, it's completely fine to keep this unit-less so long as the documentation states it clearly.

Meanwhile, look at [how Unitful.jl over in Julia-land affects performance... :sob:](https://github.com/JuliaAstro/Cosmology.jl/pull/16)